server/controllers/index.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
-    patterns: {
-        index: (ctx) => ctx.render('patterns/index', {}),
-        typography: (ctx) => ctx.render('patterns/typography', {}),
-        grids: (ctx) => ctx.render('patterns/grids', {}),
-        palette: (ctx) => ctx.render('patterns/palette', {}),
-        icons: (ctx) => ctx.render('patterns/icons', {})
-    },
-    healthcheck: (ctx) => ctx.body = 'ok'
+  patterns: {
+    index: (ctx) => ctx.render('patterns/index', {}),
+    typography: (ctx) => ctx.render('patterns/typography', {}),
+    grids: (ctx) => ctx.render('patterns/grids', {}),
+    palette: (ctx) => ctx.render('patterns/palette', {}),
+    icons: (ctx) => ctx.render('patterns/icons', {})
+  },
+  healthcheck: (ctx) => ctx.body = 'ok'
```

Comment:
This is still a lint error with standard settings ([arrow functions should not return assignment](http://eslint.org/docs/rules/no-return-assign)). Do we want to fix, or override?